src/components/Navigation/SideDrawer/SideDrawer.tsx

```diff
@@ -24,28 +24,31 @@ export const SideDrawer: FC = () => {
       <div>
         <NavLink
           to="/home"
-          className={navLinkClasses}
-          activeClassName={navLinkActiveClasses}
+          className={({ isActive }) =>
+            navLinkClasses + (isActive ? " " + navLinkActiveClasses : "")
```

Comment:
What about i.e. this?: 

```js
`${navLinkClasses} ${isActive ? navLinkActiveClasses : ''}`
```

Reply:
could do that some other time & also move out the navlink classes, they don't need to be created on every render.

Reply:
👍  wannna create an issue for that?

Reply:
If you want you can :) 
I don't think it's that urgent atm.